packages/engine/src/simulation/package/state/packages/behavior_execution/package.py

```diff
@@ -2,32 +2,40 @@
 import traceback
 
 
+def hash_behavior_id(behavior_id):
+    # `behavior_id[0]` is language index and we support fewer than 10 languages.
+    # TODO: Either keep in sync with behavior id generation in Rust part of
+    #       behavior execution package or change Rust part to generate
+    #       single numbers (rather than pairs of numbers) in the first place.
+    return behavior_id[0]*10 + behavior_id[1]
```

Comment:
```suggestion
    return (behavior_id[0] * 10) + behavior_id[1]
```

Reply:
These look equally clear to me, but sure

Reply:
_Generally_ speaking Python convention is to have spaces around operators. PEP was changed to be less clear about it unfortunately, but a lot of IDE formatting will introduce them. (And _if_ you put the spaces then I think the brackets improve readability a lot)

Reply:
Also worth saying that if we use [Black](https://github.com/psf/black) in the future then it adds spaces around it